lib/field_compactor.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+function buildRange(item) {
+  return {
+    start: item,
+    count: 1,
```

Comment:
```suggestion
    count: 1
```

No trailing-comma is used, no enforcement for that tho.

Reply:
fixed